src/util/sss_krb5.c

```diff
@@ -1104,7 +1104,6 @@ bool sss_krb5_realm_has_proxy(const char *realm)
 
     kerr = profile_get_values(profile, profile_path, &list);
     if (kerr == PROF_NO_RELATION || kerr == PROF_NO_SECTION) {
-        kerr = 0;
         goto done;
```

Comment:
False is correct, please see the commit message of cbf20090fa92cc9a6e31e3c903b21d020c519367. I don't remember why I used kerr=0 there, maybe in the past we used to return kerr.